frameworks/Java/javalin/src/main/java/com/lostsys/test/javalin/Bench.java

```diff
@@ -0,0 +1,30 @@
+package com.lostsys.test.javalin;
+
+import io.javalin.Javalin;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Bench {
+
+    public static void main(String[] args) {
+
+        Javalin app = Javalin.create().start(8080);
```

Comment:
You have `8080` hardcoded here but you also pass `8080` as an argument in the dockerfile's `CMD`.  I assume that one of these is unnecessary.